components/layout_thread_2020/lib.rs

```diff
@@ -585,7 +585,7 @@ impl LayoutThread {
             snapshot_map,
         }
     }
-
+    #[allow(clippy::too_many_arguments)]
```

Comment:
Please re-add the blank line between the two methods here.

Reply:
@mrobinson  I've added the blank lines. although I'd love to ask, other than readability, is there any other reason for the spaces?